### DISCUSSION TITLE
export saml2 metadata.xml in keycloak 23.0.5

### DISCUSSION THREAD

**Discussion by pereceptron** [keycloak/keycloak#26714] [Q&A]
Hello,
I'm new in keycloak and i'm trying to get the metadata.xml to use in my client application, and i don't find the option.
Thanks for your help!!

**Comment by phoenix-ua** [comment]
Hi,
I did not run Keycloak 23 yet, but in Keycloak 22.0.5 it was not quite straightforward how to do that.

In Keycloak 22 SAML data could be exported by clicking "SAML 2.0 Service Provider Metadata" link in Identity providers menu
![image](https://github.com/keycloak/keycloak/assets/85673694/f4ca463f-d009-49d4-9cf9-2c4605763940)

Give it a shot in version 23, maybe they did not change the UI.

**Reply by pereceptron** [reply]
Hello phoenix-ua,
thanks a lot for your answer. With it I have found it (y).
Regards
